include/deal.II/fe/fe_interface_values.h

```diff
@@ -1546,13 +1554,13 @@ class FEInterfaceValues
          const unsigned int      fe_index      = numbers::invalid_unsigned_int);
 
   /**
-   * Return a reference to the FEFaceValues or FESubfaceValues object
+   * Return a reference to the FEValuesBase object
```

Comment:
Can you mention that this is typically an FEFaceValues or FESubfaceValues except in case of using the constructor that uses two FEValuesBase pointers?